.github/workflows/update-project.yml

```diff
@@ -2,19 +2,23 @@ name: Update a package when it is updated in the monorepo
 on:
   push:
     branches:
-      - master # Every time a PR is merged to master.
+      - master # Every time a PR is merged to master
     paths:
       - 'projects/packages/**' # Only when package files are modified.
+      - 'projects/plugins/**' # Only when plugin files are modified.
 
 jobs:
-  update_package:
+  update_project:
     runs-on: ubuntu-latest
     if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
     steps:
       - uses: actions/checkout@master
-      - name: Keep Jetpack Packages up to date
+      - name: Keep Jetpack Projects up to date
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+        shell: bash -l {0}
```

Comment:
`nvm` runs on an invoked login bash shell, so we need to pass `-l` so it is invoked each time.

Otherwise, `nvm` wouldn't run the installed and set-to-use version, or in our case it'd run node 14 instead of the node 12 defined by `.nvmrc`

Reply:
Per convo on Zoom, for now, I'll take this out to match existing patterns with another PR to switch all actions to use nvm for separate review and discussion.